canonical-websites/www.ubuntu.com
# Whitepapers on the web beta

## Done
Gated whitepaper beta

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /whitepapers/robotics
- Fill in the form and read the whitepaper

## Issue / Card
Fixes https://github.com/ubuntudesign/web-squad/issues/1056
